src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -262,6 +268,25 @@ def get(self, request: Request, project: Project) -> Response:
                     else:
                         scope.set_tag("stacktrace_link.error", "file_not_found")
 
+                result["commitSha"] = None
+                githubIntegration = GitHubIntegration(integrations[0], project.organization_id)
+                try:
+                    line_no_str = ctx.get("line_no")
+                    line_no = int(line_no_str) if line_no_str else 0
```

Comment:
What is the meaning of 0 when calling git blame?

Reply:
Thank you for catching this, I removed it